src/index.html

```diff
@@ -1,13 +1,394 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>THE MET</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="icon" href="./images/logo-3x.png">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo">
+              <img src="images/logo-3x.png" alt="The Mall logo">
+            </a>
+
+            <div class="top-bar__icons">
+              <a
+                class="icon icon--phone"
+                href="tel:+1 212-535-7710, tel:+1 212-923-3700"
+                >
+              </a>
+
+              <div class="top-bar__contacts show-contacts">
+                <a href="tel:+1 212-535-7710" class="top-bar__contacts--link top-bar__contacts--fifth-avenue">The Met Fifth Avenue</a>
+                <a href="tel:+1 212-923-3700" class="top-bar__contacts--link top-bar__contacts--cloisters">The Met Cloisters</a>
+              </div>
+
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Welcome
+              <br>
+              to The MET
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo">
+            <img src="images/logo-3x.png" alt="The Mall logo">
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#">Home</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about-us">About Us</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#now-on-view">Now on view</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#explore-online">Explore Online</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contacts">Contacts</a>
+              </li>
+            </ul>
+          </nav>
+          <a href="tel:+1 212-535-7710" class="menu__phone-number-fifth-avenue">
```

Comment:
Fix format phone everywhere only in href
```suggestion
          <a href="tel:+12125357710" class="menu__phone-number-fifth-avenue">
```